config/k8s-demo.yml

```diff
@@ -7,7 +7,7 @@ lln_name: p3-lln
 # SIP configuration
 cluster_user: "fedora"
 cluster_name: "k8s-demo"
-cluster_template_name: "k8s-fedora-atomic-27"
+cluster_template_name: "k8s-fedora-atomic-29"
```

Comment:
I guess you are only upgrading the k8s demo playbook and nothing else using atomic (`config/kubernetes.yml`, `config/swarm-sip.yml`)? A hint in the commit message would be helpful :)